packages/utils/package.json

```diff
@@ -27,10 +27,10 @@
   },
   "dependencies": {
     "@kiltprotocol/types": "workspace:*",
-    "@polkadot/keyring": "^5.4.4",
-    "@polkadot/types": "^3.6.4",
-    "@polkadot/util": "^5.4.4",
-    "@polkadot/util-crypto": "^5.4.4",
+    "@polkadot/keyring": "^5.9.2",
```

Comment:
We needed to update for the current version of `system.accountInfo` which used to be used `AccountInfo` and now its `AccountInfoWithProviders`.